code/conn_ret.py

```diff
@@ -0,0 +1,79 @@
+import json
```

Comment:
please remove this file because I already incorporated the code into network_analysis.py